test/unit/config_report_importer_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class ConfigReportImporterTest < ActiveSupport::TestCase
+  setup do
+    ActionMailer::Base.deliveries = []
+  end
+
+  context 'with user owner' do
+    setup do
+      @owner = as_admin { FactoryGirl.create(:user, :admin, :with_mail) }
+      @host = FactoryGirl.create(:host, :owner => @owner)
+    end
+
+    test 'when owner is subscribed to notification, a mail should be sent on error' do
```

Comment:
Shouldn't these be in the base class test, since that's where notifications are?